core/pom.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+      <groupId>com.pingcap.tispark</groupId>
+      <artifactId>tispark-parent</artifactId>
+      <version>0.1.0-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>tispark-core</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>TiSpark Project Core</name>
+    <url>http://github.copm/pingcap/tispark</url>
+    <licenses>
+      <license>
```

Comment:
also these I think need to be gone and keep one copy in parent only.
e.g, organizations, license and etc.